linera-chain/src/chain.rs

```diff
@@ -532,30 +532,27 @@ where
                 }
             }
         }
-
-        if bundle.goes_to_inbox() {
```

Comment:
Do I understand correctly that we can remove this b/c all bundles now goe to the inbox?